Pspcl/Pspcl.Web/Controllers/StockViewController.cs

```diff
@@ -1,27 +1,42 @@
 ﻿using AutoMapper;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.EntityFrameworkCore.Metadata.Internal;
-using Pspcl.Core.Domain;
-using Pspcl.DBConnect;
-using Pspcl.Services;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Pspcl.Services.Interfaces;
 using Pspcl.Web.Models;
 
 namespace Pspcl.Web.Controllers
 {
-    
-    public class StockViewController: Controller
-    {
-        
-        private readonly IStockService _stockService;
+	public class StockViewController : Controller
+	{
+		private readonly IStockService _StockService;
         private readonly IMapper _mapper;
-        
-        [HttpGet]
-        public IActionResult AddStock ()
-        {
-         
-            return View("~/Views/StockView/AddStock.cshtml");
-        }
+        public StockViewController(IStockService addStockService)
+		{
+			_StockService = addStockService;
+		}
 
+		public IActionResult StockView()
```

Comment:
Why name is stock view ?

Reply:
Abhijeet changed this and it was changed when I took pull from develop.